src/libraries/System.Memory/src/System/Buffers/ReadOnlySequence.cs

```diff
@@ -651,8 +651,7 @@ private enum SequenceType
             MultiSegment = 0x00,
             Array = 0x1,
             MemoryManager = 0x2,
-            String = 0x3,
-            Empty = 0x4
+            String = 0x3
```

Comment:
If you leave the comma on the end of String then this diff will be smaller, and whatever the future next diff is would also be smaller...